src/UI/viewfinder/ScanningArea.tsx

```diff
@@ -89,7 +89,7 @@ const ScanningAreaContainer = styled.section`
   left: 50%;
   transform: translate(-50%, -50%);
   //-------//
-
+  outline: 1px solid hotpink;
```

Comment:
Is this fo reals or was added fo testing?

Reply:
whoops, I removed it